templates/app/scripts/offliner/safeguard.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * This script exists for your convenience. Requests to this URL are never
+ * intercepted by the generated service worker. It is here to allow you
+ * modify the behaviour of your app even when you cached unintended content.
+ */
```

Comment:
Can you explain the use case(s) for this? It seems orthogonal to the purpose of this pull request. Perhaps we can consider it in a separate PR?